docs/code-reviews/recipes/csharp.md

```diff
@@ -57,10 +57,89 @@ If you are currently using the legacy FxCop analyzers, [migrate from FxCop analy
 
 ### StyleCop analyzer
 
-The StyleCop analyzer is a nuget package (StyleCop.Analyzers) that can be installed in any of your projects. It's mainly around code style rules and makes sure the team is following the same rules without having subjective discussions about braces and spaces. Detailed information can be found here: [StyleCop Analyzers for the .NET Compiler Platform](https://github.com/DotNetAnalyzers/StyleCopAnalyzers).
+The StyleCop analyzer is a NuGet package (StyleCop.Analyzers) that can be installed in any of your projects. It's mainly around code style rules and makes sure the team is following the same rules without having subjective discussions about braces and spaces. Detailed information can be found here: [StyleCop Analyzers for the .NET Compiler Platform](https://github.com/DotNetAnalyzers/StyleCopAnalyzers).
 
 The minimum rules set teams should adopt is the [Managed Recommended Rules](https://msdn.microsoft.com/en-us/library/dd264893.aspx) rule set.
 
+### Improving Code Quality on an Existing Code Base
+
+This describes a method to follow to review and improve the quality of an existing code base. This method is independent from the programming language.
```

Comment:
Since this section is independent from the programming language, perhaps we should add a new generic recipe document. Otherwise a reader using language other than C# will miss this useful information.

Reply:
@tompaana - good point, let me take a look at how we can rearrange this to be more agnostic to programming language.

Reply:
The csharp.md updates will now be addressed in a separate PR if needed.